modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -0,0 +1,101 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-kubernetes-nmstate-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-kubernetes-nmstate-operator-CLI_{context}"]
+= Installing the Kubernetes NMState Operator using the CLI
+Instead of using the web console, you can install the Kubernetes NMState Operator by using the {product-title} CLI (oc).
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
```

Comment:
Wouldn't these be past tense since they're prerequisites? I know we often write OC CLI has past tense, anyways:

"* You have installed the OpenShift CLI (`oc`)."